### PR TITLE
[MAINT] add recent versions to github bug form

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to https://github.com/nilearn/nilearn/pull/3610#discussion_r1147401239

Changes proposed in this pull request:

- add latest version to the github form

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -62,6 +62,8 @@ body:
     # only list the 10 latest versions
     description: What version of Nilearn are you using?
     options:
+      - 0.10.1rc1
```
Comment (ymzayek) [inline .github/ISSUE_TEMPLATE/bug.yml:65]: Maybe also add 0.10.1.dev?

Comment (Remi-Gau) [inline .github/ISSUE_TEMPLATE/bug.yml:65]:
```suggestion
      - 0.10.1.dev
      - 0.10.1rc1
```

**Review by ymzayek — APPROVED** [review]
Thanks!

**Review by NicolasGensollen — APPROVED** [review]
Thanks for fixing this quickly @Remi-Gau !

**Pull request merged**